ci-automation/packages.sh

```diff
@@ -10,8 +10,6 @@
 
 # OS image binary packages build automation stub.
 #   This script will use an SDK container to build packages for an OS image.
-#   It will update the versionfile with the OS packages version built,
-#    and will add a version tag (see INPUT) to the scripts repo. 
```

Comment:
Prerequisites below probably should get an entry about the built version being recorded in versionfile.